### ISSUE TITLE
[CoE Starter Kit - QUESTION] Adding D365 apps to environment created via Environment Request app

### ISSUE THREAD

**Issue by LilianThomsen** [microsoft/coe-starter-kit#2524]
### What is your question?

We get this error when trying to install the D365 Sales Hub app in an environment created via the Environment Request app in CoE.

![image](https://user-images.githubusercontent.com/103517574/164240980-45ffaeaa-4aaa-4a75-b441-f11a369d40c1.png)

Is it not possible to create an environment for Dynamics via the Request app?

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

March 2022

### What app or flow are you having the issue with?

Maker - Environment Request

**Comment by Jenefer-Monroe** [comment]
Marking as bug for you Denise, but may be feature, I'll let you update as needed.

**Comment by denisem-msft** [comment]
Yes, this should be an option when the environment is created. We will create an interface to pass in the required values to enable these apps after validating with that team.

**Comment by denisem-msft** [comment]
This will require updates to the Power Platform for Admins connector to bring in the correct app template options that are available in your tenant. Having discussions with the engineering team to support this change and will provide update when that has moved forward.

**Comment by kbilakanti** [comment]
@denisem-msft - Can I know when is the expected timeline for this feature to be implemented and rolled out? we have few customers who require both "Enable D65 Apps" and "Deploy Sample Apps" toggle to be added as part of the Environment Creation Request from CoE App.

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
